README.rst

```diff
@@ -9,16 +9,16 @@ Contributing
 
 This project only requires that:
 
-#. you have `tox`_ installed
-#. you have Python 3 installed locally
+#. you have `sphinx`_ installed (`pip install sphinx`)
 
 To contribute to this repository, fork the project and make your changes.
-Before submitting a pull request, please check that your changes render as 
+Before submitting a pull request, please check that your changes render as
 expected by running::
 
-    $ tox -e docs,serve
+    $ make html
```

Comment:
That's fine, and I think I'd be fine closing this issue, #7 (which was merged) solves my real itch.

Just to drill down into this, my understanding is that the `make.bat` file handles the building under Windows. That said I don't really use Windows, so if it's easier to do all this with tox, that's fine.

I do argue the point about "implicitly asking them to install sphinx globally" because currently we're implicitly asking that to install *tox* globally.